src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java

```diff
@@ -86,7 +86,14 @@ public User findAdminByAccountId(Long accountId) {
     @Override
     public List<User> findAll(Pageable pageable) {
         try {
-            return jdbcTemplate.query(Queries.SQL_SELECT_ALL_USERS, userRowMapper, pageable.getPageSize(), pageable.getOffset());
+
+            String sort = pageable.getSort().toString().replaceAll(": ", " ");
+            final String selectAllUsersQuery = String.format(Queries.SQL_SELECT_ALL_USERS,
+                                                            sort,
+                                                            pageable.getPageSize(),
+                                                            pageable.getOffset());
+            System.out.println(selectAllUsersQuery);
```

Comment:
remove

Reply:
remove the whole method or just sort?

Reply:
95 line.
Or if you want to log something, use logger